modules/troubleshooting/pages/index.adoc

```diff
@@ -0,0 +1,170 @@
+
+Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed services.
+----
+
+The output shows:
+
+* A site exists in the specified namespace.
+* A link exists to two other sites.
+* A service is exposed on the {service-network}, and is accessible from this namespace.
+
+--
+
+. Check the link status:
++
+--
+----
+$ skupper link status --namespace east
+
+Link link1 is active
+----
+
+A link exists from the specified site to another site, meaning a token from another site was applied to the specified site.
+
+NOTE: Running `skupper link status` on a connected site only produces output if a token was used to create a link.   
+--
+
+. Check the {service-network}:
++
+--
+----
+$ skupper network status --namespace west
+
+Sites:
+├─ [local] 05f8c38 - west 
+│  URL: 10.110.15.54
+│  mode: interior
+│  name: west
+│  namespace: west
+│  version: 1.0.2
+│  ╰─ Services:
+│     ╰─ name: backend
+│        address: backend: 8080
+│        protocol: tcp
+╰─ [remote] 1537b82 - east 
+   URL: 10.97.26.100
+   name: east
+   namespace: east
+   sites linked to: 05f8c38-west
+   version: 1.0.2
+   ╰─ Services:
+      ╰─ name: backend
+         address: backend: 8080
+         protocol: tcp
+         ╰─ Targets:
+            ├─ name: backend-77f8f45fc8-smckp
+            ├─ name: backend-77f8f45fc8-gh6tp
+            ╰─ name: backend-77f8f45fc8-m58tg
+----
+
+The output shows:
+
+* There are two sites on the {service-network}, `east` and `west`.
+* Details for each site, for example the namespace names.
+* The original services that are exposed (Targets), in this case the three backend services exposed using the `tcp` protocol.
+* The services available on the {service-network}, including port, for example, `backend:8080`. 
+--
+
+
+// Type: procedure
+[id='creating-debug']
+== Creating a skupper debug tarball
+
+The debug tarball contains all the logs from the {skupper-name} components, and provides detailed information to help debug issues.
+
+
+
+. Create the debug tarball:
++
+--
+----
+$  skupper debug dump my-site
+
+Skupper dump details written to compressed archive:  my-site.tar.gz
+
+----
+
+--
+
+// Type:procedure
+[id='common-problems']
+== Resolving common problems
+
+The following issues and workarounds might help you debug simple scenarios when evaluating {skupper-name}.
+
+*Cannot initialize skupper*
+
+If the `skupper init` command fails, consider the following options:
+
+* Check the load balancer.
++
+--
+If you are evaluating {skupper-name} on minikube, a load balancer is not automatically created.
+Use the following command to create a loadbalancer:
+
+----
+$ minikube tunnel
+----
+
+For other Kubernetes flavors, see the documentation from your provider.
+
+--
+
+* Initialize without ingress.
++
+--
+This option will prevent other sites from linking to this site, but linking outwards is supported.
+Once a link is established, traffic can flow in either direction.
```

Comment:
Passive voice.

Reply:
the future tense in the prev sentence is prob more egregious, and preventable ;)